theme/less/_homepage.less

```diff
@@ -0,0 +1,82 @@
+body div
+{
+  grid-template-areas: 
+    "header header"
+    "main main"
+    "footer footer";
+}
+
+main article
+{
+  display: grid;
+  grid-template-columns: auto auto;
+  column-gap: 20px;
+}
+
+main article > *:first-child
+{
+  grid-column: 1 e("/") 3;
+}
+
+.box
```

Comment:
Can not figure out why widths of boxes are different. You can check out chrome DevTools size measurement tool.

Reply:
Changing `grid-template-columns: auto auto;` to `grid-template-columns: 1fr 1fr;` adjusts the sizes.

Reply:
Good job resolving this.